Cargo.toml

```diff
@@ -146,6 +146,7 @@ wasm-bindgen-futures = "0.4.33"
 js-sys = "0.3.56"
 getrandom = { version = "0.2.8", features = ["js"] }
 uuid = { version = "1.2.2", features = ["v4", "fast-rng", "serde"] }
+getaddrs = "0.1.0"
```

Comment:
`getaddrs` has a single release 6 years ago and their github link points no where

Reply:
@elichai I've removed `getaddrs` in the latest commit and replaced it with `local_ip_address` which is maintained.

Crate: https://docs.rs/local-ip-address/latest/local_ip_address/index.html
GitHub: https://github.com/EstebanBorai/local-ip-address

In addition, since the only user of this package would be kaspa-addressmanager, I've only added the crate in this package as opposed to adding it to the workspace.